tests/python/pants_test/engine/legacy/test_graph_integration.py

```diff
@@ -33,12 +33,12 @@ def use_pantsd_env_var(cls):
     'testprojects/src/python/sources:some-missing-some-not': [
       "globs('*.txt', '*.rs')",
       "Snapshot(PathGlobs(include=(\'testprojects/src/python/sources/*.txt\', \'testprojects/src/python/sources/*.rs\'), exclude=(), glob_match_error_behavior<Exactly(GlobMatchErrorBehavior)>=GlobMatchErrorBehavior(value=error), conjunction<Exactly(GlobExpansionConjunction)>=GlobExpansionConjunction(value=all_match)))",
-      'Unmatched glob: "testprojects/src/python/sources/*.rs"',
+      'Unmatched glob from testprojects/src/python/sources/BUILD for target :some-missing-some-not\'s `sources` field: "testprojects/src/python/sources/*.rs"',
```

Comment:
Open to suggestions for better wording. I don't love the `{target}'s` part because the `'s` is a little confusing to me to see, and I'd prefer that we could use `` to escape the target name.